File: Laborator/Java/AgentieTurism/src/main/java/ro/mpp2024/utils/FlightEvent.java
package ro.mpp2024.utils;

public class FlightEvent implements Event {
    public enum EventType { FLIGHTS_UPDATED }

    private final EventType type;

    public FlightEvent(EventType type) {
        this.type = type;
    }

    public EventType getType() {
        return type;
    }
}

File: Laborator/CSharp/AgentieTurism/Utils/FlightEvent.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AgentieTurism.Utils
{
    public class FlightEvent : Event
    {
        public enum EventType { FLIGHTS_UPDATED }

        public EventType Type { get; }

        public FlightEvent(EventType type)
        {
            Type = type;
        }
    }
}
